lib/index.ts

```diff
@@ -45,10 +46,25 @@ export async function inspect(root, targetFile, options: Options = {}) {
     getMetaData(root, targetFile),
     getDependencies(root, targetFile),
   ]);
-  return {
-    plugin: result[0],
-    package: result[1],
-  };
+
+  const hasDepGraph = result[1].dependencyGraph && !result[1].dependencyTree;
+  const hasDepTree = !result[1].dependencyGraph && result[1].dependencyTree;
+
+  //TODO @boost: get rid of the rest of depTree and fully convert this plugin to use depGraph
+  if(hasDepGraph) {
+    return {
+      plugin: result[0],
+      dependencyGraph: result[1].dependencyGraph
+    }
+  } else if(hasDepTree) {
+    return {
+      plugin: result[0],
+      package: result[1].dependencyTree
+    }
+  }
+
+  //TODO @boost: remove me
+  throw new Error('Failed to scan this go project.');
```

Comment:
This will be updated once we move also `golangdep` and `govendor` to use depGraph (part of another pr).
The final result expected after the ☝️ changes is the following one: 

```
return {
     plugin: result[0],
     dependencyGraph: result[1]
}
```

Where `result[1]` will contains a `DepGraph`